tests/php/test-class-amp-theme-support.php

```diff
@@ -1547,6 +1559,18 @@ public function test_filter_admin_bar_script_loader_tag( $setup_callback, $asser
 		$assert_callback( new DOMXPath( $dom ) );
 	}
 
+	/**
+	 * Test filter_admin_bar_script_loader_tag when ->deps is not an array.
+	 *
+	 * @covers \AMP_Theme_Support::filter_admin_bar_script_loader_tag()
+	 */
+	public function test_filter_admin_bar_script_loader_tag_non_array() {
+		wp_enqueue_script( 'admin-bar' );
+		$GLOBALS['wp_scripts']->registered['admin-bar']->deps = null;
+		$tag = 'example-tag';
```

Comment:
Same as above. This should be an actual script tag as is normally passed into the method.

Reply:
Sure, applied in the same commit, https://github.com/ampproject/amp-wp/pull/4207/commits/68964a40ead0dced8d50465dc09cd71788363c6b